assets/src/web-stories-block/block/components/storyCard.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { RawHTML } from '@wordpress/element';
+import { dateI18n, format, __experimentalGetSettings } from '@wordpress/date';
+
+function StoryCard({
+  title,
+  excerpt,
+  poster,
+  author,
+  date,
+  isShowingAuthor,
+  isShowingDate,
+  isShowingTitle,
+  isShowingExcerpt,
+  imageOnRight,
+}) {
+  const singleStoryClasses = classNames('web-stories-list__story-wrapper');
+  const imageAlignmentClass = classNames('web-stories-list__inner-wrapper', {
+    [`image-align-right`]: imageOnRight,
+    [`image-align-left`]: !imageOnRight,
+  });
+  const hasContentOverlay = isShowingTitle || isShowingAuthor || isShowingDate;
+  const dateFormat = __experimentalGetSettings().formats.date;
```

Comment:
Can you explain usage for experimental feature here

Reply:
I think there's no other way in Gutenberg to get the date format.

Example:

https://github.com/WordPress/gutenberg/blob/270f6eb07a243c65d60dad3ee4c3d56772ffe1bd/packages/block-library/src/latest-posts/edit.js#L450

Reply:
Okay, add a todo or another note to keep on eye on this. If they remove this experimental function.